pyvrp/cpp/LoadSegment.cpp

```diff
@@ -13,3 +13,13 @@ LoadSegment::LoadSegment(ProblemData::Client const &client, size_t dimension)
       load_(std::max<Load>(delivery_, pickup_))
 {
 }
+
+LoadSegment::LoadSegment(ProblemData::VehicleType const &vehicleType,
+                         size_t dimension)
+    : delivery_(0),
+      // Initial load is always a pickup quantity: it's already on the vehicle,
+      // and needs to be dropped off at a depot.
```

Comment:
If initial load is always a pickup, wouldn't the name `initial_pickup_load` be a better name for this? It's verbose, but this use case is relatively niche (reoptimization + pickup of goods).

Reply:
The interface is also getting more complicated with this. After #366, we should have a go at #620 and include a notebook on reoptimization and this particular use case.

Reply:
Agree, I don't really like the current docs/tutorial structure. We need to go back and fix that (and make it more prominent on the index page/README)